crypto/hash.go

```diff
@@ -118,22 +118,21 @@ func HashArgsSuite(suite abstract.Suite, args ...interface{}) ([]byte, error) {
 }
 
 // ConvertToBinaryMarshaler takes a slice of interfaces and returns
-// a slice of BinaryMarshalers.
-func convertToBinaryMarshaler(args ...interface{}) ([]encoding.BinaryMarshaler, error) {
+// a slice of BinaryMarshalers
+func ConvertToBinaryMarshaler(args ...interface{}) ([]encoding.BinaryMarshaler, error) {
```

Comment:
Will this be used externally (would make sense). If not, I think this should remain un-exported.